components/bal/voies-list.js

```diff
@@ -56,7 +56,8 @@ function VoiesList({voies, onEnableEditing, onSelect, setToRemove}) {
             actions={{
               onSelect: () => onSelect(voie._id),
               onEdit: () => onEnableEditing(voie._id),
-              onRemove: () => setToRemove(voie._id)
+              onRemove: () => setToRemove(voie._id),
+              onConvert: voie.nbNumeros ? undefined : () => setToConvert(voie._id)
```

Comment:
Le résultat sera le même, mais ça ne coûte pas grand-chose de donner une condition claire en ajoutant `=== 0`.
Il faudrait renvoyer `null` au lieu de `undefined` car `null` est utilisé pour indiquer explicitement qu'une valeur est volontairement absente ou inexistante, alors que `undefined` est généralement le résultat d'un comportement involontaire, comme une variable non déclarée.

```suggestion
              onConvert: voie.nbNumeros === 0 ? () => setToConvert(voie._id) : null
```